src/GraphQLNormalizr.js

```diff
@@ -179,12 +179,15 @@ export function GraphQLNormalizr ({
     return normalized
   }
 
+  const isInlineFragment = node => node.kind === 'InlineFragment'
```

Comment:
Please use the` INLINE_FRAGMENT` kind constant instead of `'InlineFragment'`

`const isInlineFragment = node =>  node.kind === Kind.INLINE_FRAGMENT`